src/containers/shared/components/Transaction/AMMClawback/parser.ts

```diff
@@ -6,6 +6,7 @@ export function parser(tx: AMMClawback, meta: any) {
   const holder = tx.Holder
   const amount = findAssetAmount(meta, tx.Asset, tx)
   if (tx.Flags) {
+    // @ts-expect-error - MPT is not being supported for AMM transactions until https://github.com/XRPLF/rippled/pull/5285 is merged
```

Comment:
I'm surprised that you are getting these Typescript transpiler errors in this PR. They must have been observed in prior PRs too.

Did you upgrade your ts-node version?

Reply:
This PR was included in 4.2.5:
https://github.com/XRPLF/xrpl.js/pull/2919
Which breaks the current code type-check due to MPT formatted differently.
However, MPT is not supported for AMM transactions until https://github.com/XRPLF/rippled/pull/5285 is released. So once that is merged we can make necessary changes to both xrpl.js and Explorer (edited) 


NEW

[5:34](https://ripple-enterprise.slack.com/archives/C014UHLUA6M/p1745357649129289?thread_ts=1745347737.574519&cid=C014UHLUA6M)
For now it would not break anything, other than type-check